cmake/FindPoly.cmake

```diff
@@ -143,6 +143,17 @@ if(NOT Poly_FOUND_SYSTEM)
       "${DEPS_BASE}/lib/libpicpolyxx${CMAKE_STATIC_LIBRARY_SUFFIX}")
   endif()
 
+  # If it isn't a wasm compilation
+  if(WASM STREQUAL "OFF")
+    set(POLY_PATCH_CMD_WASM "")
+  # Otherwise
+  else()
+    set(POLY_PATCH_CMD_WASM ${POLY_PATCH_CMD_WASM}
+      COMMAND
+        bash ${CMAKE_SOURCE_DIR}/cmake/deps-utils/Poly-wasm-patch.sh ${DEPS_BASE}
```

Comment:
A patch file would be better than a script in this case (see [here](https://github.com/cvc5/cvc5/blob/d01e59c13b7337e68806be72145b3e45e4cad89c/cmake/FindSymFPU.cmake#L42-L43) for an example).

Reply:
More than that, I will remove the need for any bash/patch file. I will insert an emscripten compilation flag that avoids the problem that this patch file is supposed to resolve. This approach is better because it makes sure we will have no problem with the poly lib in the future if they start using the variable that this patch comments.